rust/private/utils.bzl

```diff
@@ -296,3 +296,30 @@ def dedent(doc_string):
         # Remove the leading block of spaces from the current line
         block = " " * space_count
         return "\n".join([line.replace(block, "", 1).rstrip() for line in lines])
+
+def make_static_lib(ctx, rlib_file):
```

Comment:
Nit: Can you move this to [rust/private/toolchain_utils.bzl](https://github.com/bazelbuild/rules_rust/blob/8cca226c6bc3c55444c98a6a08e7d19b6dd028f2/rust/private/toolchain_utils.bzl)?

Reply:
Just to clarify: this is referenced from both [rust/toolchain.bzl](https://github.com/bazelbuild/rules_rust/blob/8cca226c6bc3c55444c98a6a08e7d19b6dd028f2/rust/toolchain.bzl) and [rust/private/rustc.bzl](https://github.com/bazelbuild/rules_rust/pull/825/files#diff-ecce7f6cdb4bc2f884f23fccb9a20174ad2b357dc6745314054e5eb8b784d2e0R636), so moving it to rust/private/toolchain_utils.bzl would require adding a dependency for rust/private/rustc.bzl on rust/private/toolchain_utils.bzl, which seems a bit off. Is that expected?

Reply:
Ah, I missed that it was in `rustc.bzl`, disregard!